### DISCUSSION TITLE
[deepin-unioncode] about filebrowser and project

### DISCUSSION THREAD

**Discussion by Yurii-huang** [linuxdeepin/developer-center#5966] [Developer Discussion | 开发者讨论]
Suggest reducing the occupation of file fd. 
The so-called file browsing is already included in the project tree displayed in the file, but it is actually irrelevant. 
This is a strong community suggestion, at least some people have reported that file engineering cannot operate on files, which is very strange.

**Comment by BLumia** [comment]
The file manager itself doesn't take a large amount of fds, I'm not quite sure what is the goal that we should be work on. We might need some detailed information to know what's the exact issue here.

> some people have reported that file engineering cannot operate on files

Could you provide some links related to this? It would be helpful if you could provide some context for the discussion.

**Reply by Yurii-huang** [reply]
sorry，This source is deepin-unioncode
